### ISSUE TITLE
Expose Sui RPC on port 9000

### ISSUE THREAD

**Issue by heyitaki** [wormhole-foundation/wormhole#2019]
Sui RPC is currently being served on port 9002 in Tilt, but apps such as the Sui wallet and Sui explorer expect RPC to be on port 9000. 

Currently `Tiltfile` port forwards 9000 as Solana PubSub, but a quick look at `solana-devnet.yaml` ([link](https://github.com/wormhole-foundation/wormhole/blob/048b8834c9a79bdd882c10c23272303593e6be35/devnet/solana-devnet.yaml#L86)) suggests that nothing is being exposed on port 9000.  As such, the RPC should be properly exposed on port 9000; this should be a small change given that this port was not actually previously being used.

**Comment by heyitaki** [comment]
This should also fix the following warning that is popping up in Tilt. Because `solana-devnet.yaml` does not expose anything on port 9000 as specified in the Tiltfile, Tilt automatically forwards the first port in the list (in this case, 8001).
```
solana-devnet │ WARNING: k8s_resource(name='solana-devnet', port_forwards='9000') currently maps localhost:9000 to port 8001 in your container.
solana-devnet │ A future version of Tilt will change this default and will map localhost:9000 to port 9000 in your container.
solana-devnet │ To keep your project working, change your Tiltfile to k8s_resource(name='solana-devnet', port_forwards='9000:8001')
```

**Comment by heyitaki** [comment]
Resolved by #2068.
